expressions/table_generator/GeneratorFunction.hpp

```diff
@@ -0,0 +1,119 @@
+/**
+ *   Copyright 2016 Pivotal Software, Inc.
```

Comment:
Updated with `Copyright 2016, Quickstep Research Group, Computer Sciences Department, University of Wisconsin—Madison.`